scripts/qesap/test/e2e/test.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -e
+
+test_step () {
+  echo "##############################"
+  echo "# $1"
+  echo "##############################"
+}
+
+test_die () {
+  echo "DIE : $1"
```

Comment:
Would be fine, if this always includes return code of the failed command  `$?`

`echo "DIE($?)  :  $1"`